test/scale-infra/deploy_ks_core.yaml

```diff
@@ -0,0 +1,104 @@
+---
+- hosts: masters
+  remote_user: ubuntu
+  become: yes
+  gather_facts: no
+  vars:
+    ks_release:  "{{ ks_release }}"
+  tasks:
+    - name: install jq and make
+      shell: |
+             sudo apt update
+             sudo apt install jq make -y
+
+    - name: install golang
+      shell: |
+             curl -LO https://go.dev/dl/go1.23.5.linux-amd64.tar.gz
```

Comment:
The latest release of Go is 1.24.0. The latest patch release for 1.23 is 6.

Reply:
Fixed with the latest go release.